sdk/communication/communication-rooms/test/public/roomsClient.spec.ts

```diff
@@ -128,6 +128,23 @@ describe("RoomsClient", function () {
       await client.updateRoom(roomId, options);
     });
 
+    it("successfully list rooms", async function () {
+      const listRoomsResult = await client.listRooms({});
+
+      assert.isDefined(listRoomsResult);
+      assert.isNotEmpty(listRoomsResult);
+      for await (const roomModel of listRoomsResult) {
+        if (roomModel) {
+          assert.isDefined(roomModel);
+          assert.isNotEmpty(roomModel.id);
+          assert.isNotEmpty(roomModel.validFrom.toString());
+          assert.isNotEmpty(roomModel.validUntil.toString());
+        }
+
+        break;
+      }
```

Comment:
I think this test will pass even if 0 rooms are returned. Maybe add a counter to check the number of rooms returned; like the participant tests.

Reply:
Its possible that listRooms won't return any value. We can also ensure once the room is returned, it comes with values as defined in RoomModel.